Make hvac_action optional on ulm_translation_hvac

mpeterson opened the pull request:
Some climate entities do not have the hvac_action attribute and in those
cases the translation ends up with '(undefined)' as the mode part. This
is undesirable, and thus this PR removes it from the translation in case
there is no hvac_action attribute.

basbruss commented:
@mpeterson 
PR's for official cards should be directed to the `release branch` as stated in the `contribution.md`

mpeterson commented:
@basbruss Thanks, I didn't know. I did look for a `CONTRIBUTE.md` file but couldn't find any and now after you told me looked again and still can't find any. Are you sure there is one?

In any case I rebased and changed base branch.

mpeterson commented:
@basbruss  found it in `.github` ! I'll read it now

basbruss commented:
> I did look for a `CONTRIBUTE.md` file but couldn't find any and now after you told me looked again and still can't find any. Are you sure there is one?

It is located in the `.github` folder: https://github.com/UI-Lovelace-Minimalist/UI/blob/main/.github/CONTRIBUTING.md
And there are also references underneath a PR added by Github.
<img width="637" alt="image" src="https://user-images.githubusercontent.com/68892092/180153309-c0f6d70a-573c-4a42-a92b-c0659799dd41.png">

Thanks for your contributions 😉

mpeterson commented:
> > I did look for a `CONTRIBUTE.md` file but couldn't find any and now after you told me looked again and still can't find any. Are you sure there is one?
> 
> It is located in the `.github` folder: https://github.com/UI-Lovelace-Minimalist/UI/blob/main/.github/CONTRIBUTING.md And there are also references underneath a PR added by Github. <img alt="image" width="637" src="https://user-images.githubusercontent.com/68892092/180153309-c0f6d70a-573c-4a42-a92b-c0659799dd41.png">
> 

Oh, wow. I would have never seen that small information block from GitHub. Really bad placement there :) 

> Thanks for your contributions 😉

for sure! Thanks to you and all the other maintainers. I really enjoy using the project.
